.github/workflows/docs.yaml

```diff
@@ -1,6 +1,10 @@
 name: Build & deploy adaptor documentation
 
 on:
+  workflow_run:
```

Comment:
@josephjclark what does this logic do ?

Reply:
Oh don't worry about this, it's entirely unrelated.

It just ensures that docs don't get build until after a release is published